independent-publisher-connectors/Influenza and Covid-19/apiProperties.json

```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#f7f7f7",
+    "capabilities": [],
+    "publisher": "Kevin Comba Gatimu , Denis Wachira Kathuri",
```

Comment:
@kelcho-spense Please remove the space after `"Kevin Comba Gatimu"`.

Reply:
I have rectified the error.